### ISSUE TITLE
Wallpaper does not re-start after PC Wakes from Sleep

### ISSUE THREAD

**Issue by IAmOrion** [rocksdanister/lively#1335]
**Describe the bug**
When my computer wakes up from sleep, the wallpaper is paused.  I have to either restart Lively OR, I can simply select a different wallpaper (any will work) which appears to give it a kick start or something then just reselect my chosen wallpaper.  This happens no matter what wallpaper I choose, so I don't believe it's specific to the chosen wallpaper itself

**To Reproduce**
1. Put computer to sleep
2. Wake computer up

**Expected behavior**
Wallpaper animation to resume.

**Desktop (please complete the following information):**
 - OS: Windows 11
- Matrix Rain Customizable - although it will happen regardless of what wallpaper I choose.

**Additional context**
Sidebars / widget bars etc are disabled

**Log file (Important)**
[20220801_110844.txt](https://github.com/rocksdanister/lively/files/9240845/20220801_110844.txt)

**Comment by rocksdanister** [comment]
1. Can this be reproduced always? if you sleep and wake up system immediately is it still paused?
2. Are you sure its paused? Do you have `Desktop Picture` enabled in `Settings -> Wallpaper`?
3. Try setting All pause settings (Application fullscreen, Application focused ..etc) to `Nothing` in `Settings -> Performance` and see if the issue is still happening.

**Comment by IAmOrion** [comment]
1) Yes, this always happens.  I just double checked putting PC to sleep, then immediately waking it - it's still Paused.
![ScreenShot-02-08-2022-10 43 36](https://user-images.githubusercontent.com/3501380/182344872-5bcd65af-6d9a-42f8-b215-cc5afeb2425c.png)
(I appreciate a screenshot would be static anyways ha - BUT, yeah, the wallpaper is paused/not animating)

2) Desktop Picture is OFF (should it be on then?)

3) Setting all to nothing didn't fix it, however....

Edit: 

When playing with the Pause settings, "Pause Algorithm" was on "All Process".  So I set my Pause settings back to what they were, then I changed "Pause Algorithm" to "Foreground Process" and it works now.  I must've changed that setting by mistake at some point not knowing what it did! (I'm an idiot)
